rest/endpoints/auth/authentication.yml

```diff
@@ -53,7 +55,7 @@ components:
           properties:
             grant_type:
               type: string
-              example: password
+              example: "password"
               enum: [password]
```

Comment:
Why do we need this?

Reply:
Just an example of value, the is shown in schema description.

Reply:
removed